docs/api/java/index.md

```diff
@@ -0,0 +1,12 @@
+# MXNet - Java Inference API
+
+MXNet supports Java for performing inference on a trained model. The MXNet Java Inference API is an extension of the [Scala Infer API](../../api/scala/infer.html) which provides model loading and inference functionality.
+
+The goal of the MXNet Java package is to provide an efficient and easy to use inference API.
+The MXNet Java package makes it easy to quickly deploy an existing model into a production level Java ecosystem.
+
```

Comment:
Since you have a dedicated install page, why not add it here?
```suggestion

## Installation
* [MXNet Java Inference API setup instructions](../../install/java_setup.md)

```